sklearn/utils/estimator_checks.py

```diff
@@ -355,10 +358,29 @@ def _is_32bit():
     return struct.calcsize('P') * 8 == 32
 
 
+def maybe_pairwise(X, estimator, kernel=linear_kernel):
```

Comment:
docstring? should this be a private function?

Reply:
Maybe the function should be called ``create_pairwise_data``? or ``maybe_pairwise_data``?